static/install

```diff
@@ -19,30 +21,32 @@ Bold_Green='\033[1;32m' # Bold Green
 Bold_White='\033[1m'    # Bold White
 
 error() {
-    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    printf "%berror%b: %s\n" "${Red}" "${Color_Off}" "$*" >&2
     exit 1
 }
 
 info() {
-    echo -e "${White}$* ${Color_Off}"
+     printf "%b%s %b\n" "${White}" "$*" "${Color_Off}"
 }
 
 info_bold() {
-    echo -e "${Bold_White}$* ${Color_Off}"
+    printf "%b%s %b\n" "${Bold_White}" "$*" "${Color_Off}"
 }
 
 success() {
-    echo -e "${Green}$* ${Color_Off}"
+    printf "%b%s %b\n" "${Green}" "$*" "${Color_Off}"
 }
 
 tildify() {
-    if [[ $1 = $HOME/* ]]; then
-        local replacement=\~/
-
-        echo "${1/$HOME\//$replacement}"
-    else
+    case "$1" in
+    "$HOME"/*)
+        tilde_replacement=\~/
+        echo "$1" | sed "s/$HOME/$tilde_replacement/g"
```

Comment:
Does that work? If  we have `s/exp/replacement/` and `exp` contains slashes as it is fairly common this will break. I suggest using `s|exp|replacement|` or `s#exp#replacement#` instead.